speech/tests/unit/test__gax.py

```diff
@@ -16,6 +16,9 @@
 
 import mock
 
+from google.cloud import _helpers
+from google.cloud.gapic.speech.v1.speech_client import SpeechClient
```

Comment:
Please don't break our grand tradition of keeping imports (of modules under test) at runtime rather than import time.

ISTM you are doing it just to have shorter lines in `mock.patch` expressions below. Not the most virtuous cause?